packages/hardhat-core/package.json

```diff
@@ -23,21 +23,22 @@
     "hardhat": "internal/cli/bootstrap.js"
   },
   "scripts": {
-    "lint": "yarn prettier --check && yarn eslint",
-    "lint:fix": "yarn prettier --write && yarn eslint --fix",
+    "lint": "pnpm prettier --check && pnpm eslint",
+    "lint:fix": "pnpm prettier --write && pnpm eslint --fix",
     "eslint": "eslint 'src/**/*.ts' 'test/**/*.ts'",
     "prettier": "prettier \"**/*.{js,md,json}\"",
-    "prebuild": "cd ../../crates/edr_napi && yarn build",
-    "prebuild:tracing": "cd ../../crates/edr_napi && yarn build:tracing",
-    "build": "tsc --build --force --incremental .",
-    "build:tracing": "tsc --build --force --incremental .",
-    "pretest": "cd ../.. && yarn build",
+    "pretest": "cd ../.. && pnpm build",
     "test": "mocha --recursive \"test/**/*.ts\" --exit",
     "test:except-provider": "mocha --recursive \"test/**/*.ts\" --exclude \"test/internal/hardhat-network/provider/**/*.ts\" --exit",
     "test:except-tracing": "mocha --recursive \"test/**/*.ts\" --exclude \"test/internal/hardhat-network/stack-traces/**/*.ts\" --exit",
     "test:provider": "mocha --recursive \"test/internal/hardhat-network/provider/**/*.ts\" --exit",
     "test:tracing": "mocha --recursive \"test/internal/hardhat-network/stack-traces/**/*.ts\" --exit",
-    "prepublishOnly": "yarn build",
+    "test:forking": "mocha --recursive \"test/internal/hardhat-network/{helpers,jsonrpc,provider}/**/*.ts\"",
```

Comment:
```suggestion
```

This was removed in `edr/main`, in favour of `test:provider`, `test:tracing`, test:except-provider`, and `test:except-tracing`

Reply:
Done, thanks!